### PR TITLE
change oopjit flag name; misc cleanup for shutdown from console host

### PR DESCRIPTION
**Pull request by MikeHolman**

### CODE REVIEW

**Comment by agarwal-sandeep** [comment]
>         RpcExcept(RpcExceptionFilter(RpcExceptionCode()))

nit: indentation 

---
Refers to: lib/JITClient/JITManager.cpp:265 in 9f08b02. [](commit_id = 9f08b021cf28790a360128c51db89d2fdd1021f3, deletion_comment = False)

File: lib/JITClient/JITManager.cpp [hunk 1]
```diff
@@ -267,14 +268,21 @@ JITManager::CleanupProcess()
...
 HRESULT
 JITManager::Shutdown()
 {
+    // this is special case of shutdown called when runtime process is a parent of the server process
+    // used for console host type scenarios
     HRESULT hr = S_OK;
     Assert(IsOOPJITEnabled());
+    Assert(m_rpcBindingHandle != nullptr);
+
+    CleanupProcess();
```
Comment (agarwal-sandeep) [inline lib/JITClient/JITManager.cpp]: Is a failure in closing indicates runtime issue? is so should we assert?

**Comment by agarwal-sandeep** [comment]
:shipit:

File: lib/JITClient/JITManager.cpp (hunk 1 above)
Comment (MikeHolman) [inline lib/JITClient/JITManager.cpp]: Sure, I can make this an assert.

**Comment by MikeHolman** [comment]
I have discussed this offline with @curtisman, who suggested not enabling by default in ch, as oop jit isn't currently a primary scenario for ChakraCore. I still want the other changes though (e.g. renaming -enableoutofprocjit to -oopjit) so I'll change the review to reflect this.

**Pull request merged**